lib/cdo/metrics_helper.rb

```diff
@@ -11,3 +11,16 @@ def self.write_metric(name, metadata, value, timestamp=nil)
     dataset.insert(data)
   end
 end
+
+# Summary of the data currently being logged in the metrics table:
+# -----------------------------------------------------------------------------
+# |       name        |          metadata           |          value           |
+# -----------------------------------------------------------------------------
+# |    dtt_start      |        commit hash          |  0 - if automatic        |
+# |                   |                             |  1 - if manual           |
```

Comment:
A few file-level questions/comments:
1. the values of 0 and 1 for automatic and manual should really be defined as constants in this file
2. Does the "value" column have to be a number? It would be helpful to add some documentation for the parameters to `self.write_metric`, especially since there is no easy way to see the definition of the underlying mysql table.

Reply:
The value column is a double.  Guidance from Will and Jeremy is that the data table should be as versatile as possible so we can chuck in any type of metric we want to measure; the columns are intentionally very general with the idea that metadata is flexible enough to hold whatever information makes sense. 
If this works, then I'll write up some documentation outlining the structure and how to add new metrics. Where should that info live? The wiki? Part of the README?

Reply:
The `write_metric` method in this file should definitely have documentation including (1) what this method does and (2) the types of each argument. Then, some high-level documentation pointing to this method could either go in the wiki (if the audience is developers) or in the github readme (if the audience is developers + contributors).

Reply:
I updated to include documentation for the `write_metric` method consistent with methods in other lib/cdo files.

Reply:
The new docs look great!